README.md

```diff
@@ -6,40 +6,32 @@ A collection of small widgets, easy to drop into place as Ember Components.
 ## Demo and Documentation
 http://addepar.github.com/ember-widgets/
 
-
-## Getting started
-
 ### JS Bin Starter Kit
 
 http://emberjs.jsbin.com/hebexa/1/edit
 
-### Installation
-
-With bower: `bower install ember-widgets --save`
-
-Or, just include `dist/js/ember-widgets.js` and `dist/css/ember-widgets.css` in your app.
+## Ember-CLI Installation
```

Comment:
I think this section should be `npm install --save-dev ember-widgets`, as we should put "how to use" instructions at the top of the app. The `git clone...` instructions should be father down in the README, in a "devloping" section.